src/notation/view/widgets/editstyle.ui

```diff
@@ -8106,6 +8106,77 @@
                   </spacer>
                  </item>
                 </layout>
+               </item>
+                              <item>
+                <layout class="QHBoxLayout" name="horizontalLayout_17">
+                 <item>
+                  <widget class="QLabel" name="minHangingTieLengthLabel">
+                   <property name="sizePolicy">
+                    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
+                     <horstretch>0</horstretch>
+                     <verstretch>0</verstretch>
+                    </sizepolicy>
+                   </property>
+                   <property name="text">
+                    <string>Minimum hanging tie length:</string>
+                   </property>
+                   <property name="buddy">
+                    <cstring>minHangingTieLength</cstring>
+                   </property>
+                  </widget>
+                 </item>
+                 <item>
+                  <widget class="QDoubleSpinBox" name="minHangingTieLength">
```

Comment:
It would be good to fix the keyboard navigation order too, to prevent a situation like https://github.com/musescore/MuseScore/issues/25530; see that issue for info